### ISSUE TITLE
Hiding bodies is not realistic

### ISSUE THREAD

**Issue by ghost** [DayZMod/DayZ#586]
As a suggestion, I don't think that players should be able to 'hide' bodies using the current method.
Being able to drag, bury, burn, blow or cut them up (:S) would be far more realistic..  

Removing/replacing the option in such a way that means is more difficult to dispose of bodies would add extra gameplay dynamics, e.g. the ability to move the body and stash it for the re-spawning victim to recollect their gear, or leave it where it is in order to draw in other players, etc.

I remember playing years back and some of the most compelling and engrossing parts were finding dead bodies.  We don't see that so much now.

**Comment by PatriotCZ** [comment]
Yes, exactly. I was thinking about the same. Being able to drag them would not be a hard task to do in my opinion. ArmA already provides some of the scripts and modules that could be used. Also I would add feature that bodies could not be hidden (buried) into an asphalt or concrete, so if you see a dead body in barracks you know someone was there recently.

**Comment by R4Z0R49** [comment]
Its not something we are planning to change just now maybe at a later date.

Thanks

**Comment by rvett** [comment]
I realize that this has been commented on by R4Z0R49, but I'd love to see the "hide body" option replaced by "drag body" as well. So this is an attempt at raising the priority of this suggestion.

**Comment by KPanicEKS** [comment]
Or maybe you can only hide zombies or players if you have a shovel or
entrenching tool and the bodies disappear after a few minutes instead of
instantly.
